gstools/field/summator.pyx

```diff
@@ -12,31 +12,26 @@ from libc.math cimport sin, cos
 cimport numpy as np
 
 
-DTYPE = np.double
-ctypedef np.double_t DTYPE_t
-
-
 def summate(
-    const double[:,:] cov_samples,
+    const double[:, :] cov_samples,
     const double[:] z_1,
     const double[:] z_2,
-    const double[:,:] pos
+    const double[:, :] pos
     ):
-    cdef int i, j, d, X_len, N
+    cdef int i, j, d
     cdef double phase
-    cdef int dim
-    dim = pos.shape[0]
+    cdef int dim = pos.shape[0]
 
-    X_len = pos.shape[1]
-    N = cov_samples.shape[1]
+    cdef int X_len = pos.shape[1]
+    cdef int N = cov_samples.shape[1]
 
-    cdef double[:] summed_modes = np.zeros(X_len, dtype=DTYPE)
+    cdef double[:] summed_modes = np.zeros(X_len)
 
     for i in prange(X_len, nogil=True):
         for j in range(N):
             phase = 0.
             for d in range(dim):
-                phase += cov_samples[d,j] * pos[d,i]
+                phase += cov_samples[d, j] * pos[d, i]
```

Comment:
Unfortunately `black` is still not able to deal with `*.pyx` files: https://github.com/psf/black/issues/359